ivadomed_singularity_master.def

```diff
@@ -0,0 +1,71 @@
+BootStrap: docker
+From: python:3.8-slim-bullseye
+
+%setup
+    # Anything here are executed on the BUILD HOST (not the running host)
+    # wget https://raw.githubusercontent.com/ivadomed/ivadomed/master/requirements_dev.txt
+    # wget https://raw.githubusercontent.com/ivadomed/ivadomed/master/requirements.txt
```

Comment:
[Docs](https://docs.sylabs.io/guides/latest/user-guide/definition_files.html#setup) suggest avoiding this section. I guess we should remove it altogether.